examples/with-universal-configuration-build-time/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [
+    "next/babel"
+  ],
+  "plugins": [
+    ["transform-define", "./env-config.js"]
```

Comment:
This won't work with Next 6. It has to be changed to `.babelrc.js` and the `env-config.js` file should be inlined

Reply:
committed and fixed with 2557ef1cef77e5571d1f47689606df54f6d9157d